ldms/src/store/store_csv_common.c

```diff
@@ -913,10 +899,8 @@ void close_store_common(struct csv_store_handle_common *s_handle, struct csv_plu
 	}
 
 	rename_output(s_handle->filename, FTYPE_DATA, s_handle, cps);
-	if (s_handle->altheader)
-		rename_output(s_handle->headerfilename, FTYPE_HDR, s_handle, cps);
-	if (s_handle->typeheader)
-		rename_output(s_handle->typefilename, FTYPE_KIND, s_handle, cps);
+	rename_output(s_handle->headerfilename, FTYPE_HDR, s_handle, cps);
+	rename_output(s_handle->typefilename, FTYPE_KIND, s_handle, cps);
```

Comment:
This hunk does not appear to be related to this PR.

Reply:
good catch. that's a reversal of a previous PR, and I have no idea how it got there. Must have been an error in rebasing. will fix.